app/src/main/java/com/example/android/codelabs/navigation/FlowStepFragment.kt

```diff
@@ -37,10 +38,10 @@ class FlowStepFragment : Fragment() {
 
         val flowStepNumber = arguments?.getInt("flowStepNumber")
 
-        // TODO STEP 8 - Use type-safe arguments - remove previous line!
-//        val safeArgs: FlowStepFragmentArgs by navArgs()
-//        val flowStepNumber = safeArgs.flowStepNumber
-        // TODO END STEP 8
+        //  STEP 8 - Use type-safe arguments - remove previous line!
+        val safeArgs: FlowStepFragmentArgs by navArgs()
```

Comment:
The safeArg data can be used by Fragments to access the arguments passed to them. Try commenting out line 39 and uncommenting line 43 to see them work